extensions/sonarcloud/vss-extension.json

```diff
@@ -39,7 +39,7 @@
       "uri": "https://about.sonarcloud.io/"
     },
     "getstarted": {
-      "uri": "http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html"
+      "uri": "https://aka.ms/sonarcloudlab"
     },
     "support": {
       "uri": "https://www.sonarqube.org/community/"
```

Comment:
Not sure when actual release is planned. If next week, then I could update support links to our upcoming community forum.